Anlab.Mvc/Views/Order/Index.cshtml

```diff
@@ -62,27 +86,68 @@
 @section AdditionalStyles
 {
     @{ await Html.RenderPartialAsync("_DataTableStylePartial"); }
+    <link rel="stylesheet" type="text/css" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css" />

```

Comment:
@laholstege We usually use the bootstrap datepicker instead of bringing in the whole jqueryUI library.  The change probably won't be too hard. https://github.com/uxsolutions/bootstrap-datepicker.  There's a CDN you can use to grab the min css and javascript for that project that they mention in the readme.